src/net/db.rs

```diff
@@ -1772,49 +1832,73 @@ mod test {
         .unwrap();
 
         // epoch 2.0
-        let n5 =
-            PeerDB::get_random_neighbors(db.conn(), 0x9abcdef0, 0x00, 5, 23455, false).unwrap();
+        let n5 = PeerDB::get_random_neighbors(db.conn(), 0x9abcdef0, 0x00, 0x00, 5, 23455, false)
```

Comment:
In these tests, `peer_version` would be something like `0x18000009` (for epoch 2.4).